js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -381,7 +386,7 @@ const AssetGraphExplorerWithData = ({
       focusGroupIdAfterLayoutRef.current &&
       layout.groups[focusGroupIdAfterLayoutRef.current]?.expanded
     ) {
-      zoomToGroup(focusGroupIdAfterLayoutRef.current, false);
+      zoomToGroup(focusGroupIdAfterLayoutRef.current, false, false);
```

Comment:
Not a fan of boolean args, looks like it's been this way for a while though so I won't insist that you change them.